code/Model/Submission/SubmittedFileField.php

```diff
@@ -41,25 +42,22 @@ public function getFormattedValue()
     {
         $name = $this->getFileName();
         $link = $this->getLink(false);
-        $title = _t(__CLASS__ . '.DOWNLOADFILE', 'Download File');
-        $message = _t(__CLASS__ . '.INSUFFICIENTRIGHTS', 'You don\'t have the right permissions to download this file');
-        $file = $this->getUploadedFileFromDraft();
-
         if ($link) {
-            if ($file->canView()) {
-                return DBField::create_field('HTMLText', sprintf(
-                    '%s - <a href="%s" target="_blank">%s</a>',
-                    htmlspecialchars($name, ENT_QUOTES),
-                    htmlspecialchars($link, ENT_QUOTES),
-                    htmlspecialchars($title, ENT_QUOTES)
-                ));
-            } else {
-                return DBField::create_field('HTMLText', sprintf(
-                    '<i class="icon font-icon-lock"></i> %s - <em>%s</em>',
-                    htmlspecialchars($name, ENT_QUOTES),
-                    htmlspecialchars($message, ENT_QUOTES)
-                ));
+            $title = _t(__CLASS__ . '.DOWNLOADFILE', 'Download File');
+            $file = $this->getUploadedFileFromDraft();
+            $formattedMessage = '';
+            if (!$file->canView()) {
```

Comment:
I still see this message if I logged in as "Content author". If I'm "ADMIN" everything is ok.